lib/apr_web/live/order_dashboard_live.ex

```diff
@@ -39,109 +34,25 @@ defmodule AprWeb.OrderDashboardLive do
   end
 
   def mount(_session, socket) do
-    if connected?(socket), do: AprWeb.Endpoint.subscribe("events")
-
-    {:ok, get_events(socket)}
-  end
-
-  def handle_event("today", _, socket) do
-    {:noreply, get_events(socket, 1)}
-  end
-
-  def handle_event("last_week", _, socket) do
-    {:noreply, get_events(socket, 7)}
-  end
-
-  def handle_event("last_month", _, socket) do
-    {:noreply, get_events(socket, 30)}
-  end
+    if connected?(socket) do
+      AprWeb.Endpoint.subscribe("events")
+      send(self(), :initialize)
+    end
 
-  def handle_info(%{event: "new_event", payload: _event}, socket) do
-    {:noreply, get_events(socket)}
+    {:ok, assign(socket, events: [], totals: %{amount_cents: 0, commission_cents: 0})}
```

Comment:
changed `mount` to only create empty state and call https://github.com/artsy/apr-dashboard/pull/13/files#diff-4fd30bff4fce07f3e8d83ccc7fcc65d7R39 separately so the html page renders and then we populate it separately